docs/source/ddp_comm_hooks.rst

```diff
@@ -2,12 +2,15 @@ DDP Communication Hooks
 =======================
 
 DDP communication hook is a generic interface to control how to communicate
-gradients across workers.
+gradients across workers by overriding the vanilla allreduce in ``DistributedDataParallel``.
```

Comment:
nit: Might be nice to have a clickable link to DDP here: https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html#torch.nn.parallel.DistributedDataParallel.